enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/catchup/storecopy/RemoteStore.java

```diff
@@ -50,7 +50,7 @@
  */
 public class RemoteStore
 {
-    private static final Supplier<TerminationCondition> DEFAULT_TERMINATION_CONDITIONS = () -> new MaximumTotalRetries( 10, 10 );
+    private static final Supplier<TerminationCondition> DEFAULT_TERMINATION_CONDITIONS = () -> new MaximumTotalRetries( 10, 100 );
```

Comment:
Could be nice to base a pause on ExponentialBackoffStrategy.
I think the value has to be a configuration, even though it should be uncommon to want to tweak it, it is better than a hard-coded value.